### DISCUSSION TITLE
How  to: List monitor's all available resolutions and refresh rate

### DISCUSSION THREAD

**Discussion by sujoshua** [hyprwm/Hyprland#5043] [Q&A] [answered]
The command `hyprctl monitors all` only shows the current resolution and refresh rate. How can I list all the resolutions and refresh rates available?

Checking manually the content of the file `/sys/class/drm/card1-HDMI-A-1/modes` lists the resolutions, but not the refresh rates. 
`xrandr` can do this. But it is in a X11 way.

I guess there should be an easy command similar to `hyprctl monitors all` which would do that?

**Comment by vaxerski** [comment, marked as answer]
`wlr-randr` can show you that, or hyprctl monitors in `-git`.
